docs/shared-content-source/docs/modules/develop/pages/use-flink-streamlets.adoc

```diff
@@ -115,3 +115,21 @@ PVCs are automatically provisioned for each Cloudflow application.
 The volumes are claimed for as long as the Pipeline application is deployed, allowing for seamless re-deployment, upgrades, and recovery in case of failure.
 
 Flink's runtime encodes all state and writes them into checkpoints. And since checkpoints in Flink offer an exactly-once guarantee of semantics, all application state are preserved safely throughout the lifetime of the streamlet. In case of failures, recovery will be done from checkpoints, and there will be no data loss.
+
+== Flink local execution
+
+If you are testing Flink streamlets locally <<Using the Local Sandbox>> you should be aware
+of several things:
+
+* Restartability. Unlike cluster deployment, local Flink execution will not restart in the case of crashes.
+* Checkpointing. By default, in the case of local runs, checkpointing is done in memory, which means that in the case of local runs, checkpointing does not survive an individual execution.
+
+The checkpointing behaviour in this case (along with many other parameters) can be overwritten
+using <<The Cloudflow Configuration Model>>. The default configuration for Flink is https://github.com/apache/flink/blob/master/flink-dist/src/main/resources/flink-conf.yaml[here].
+A complete documentation of configuration parameters, can be found https://ci.apache.org/projects/flink/flink-docs-stable/ops/config.html[here].
+
+In addition to "standard" Flink parameters, you can enable the Web UI
+when using `runLocal`. If the parameter `local.web` is set in the configuration
+(for the Flink runtime or a particular streamlet), the Flink execution is started with the Web UI, as shown
```

Comment:
Does the "as shown" refer to the cross reference that follows? If so, "as shown in"